SPECS/python-tensorflow-io-gcs/python-tensorflow-io-gcs.spec

```diff
@@ -0,0 +1,50 @@
+%global pypi_name tensorflow-io-gcs
+%global _description %{expand:
```

Comment:
I think you should dump this Macro.. Copy this content to the  %description section.

Also...I would change the Summary: below to say simply "Extended TensorFlow File Formats"

Reply:
Maybe the summary is just whatever the definition of gcs is,